### PR TITLE
added api calls and tables for tremor and trajectories

### PR DESCRIPTION
**Pull request by Scottamattic**
help is this right?

Also needs tests, but there's one week left in class, i'm not even sure this is strictly necessary.

### CODE REVIEW

File: database/database/database.py [hunk 1]
```diff
@@ -143,6 +143,71 @@ def find_accel_by_id(db, _id):
 
     return single_row_to_dict(result)
 
+def find_trajectories_by_id(db, _id):
```
Comment (snelson44) [inline database/database/database.py:146]: should be singular, not plural

File: database/database/database.py [hunk 2]
```diff
@@ -593,6 +708,68 @@ def insert_accel(db, accel_id, description, patient_id, x, y, z):
 
     return True
 
+def insert_trajectories(db, traj_id, description, patient_id, x, y, z):
```
Comment (snelson44) [inline database/database/database.py:711]: should be singular for clarity

File: database/database/database.py [hunk 3]
```diff
@@ -1807,6 +2178,56 @@ def find_one_accel(db):
...
+    Args:
+        db (slqalchemy.engine): the database engine
+
+    Returns:
+        result(dict): Dict with column names as keys. If no row,
+            an empty dict is returned.
+    """
+    logger = logging.getLogger(__name__)
+    validate_db(db, 'engine_db')
+    logger.info("Fetching random row from tremo table")
```
Comment (snelson44) [inline database/database/database.py:2218]: *tremor, typo

File: database/database/database.py [hunk 4]
```diff
@@ -1807,6 +2178,56 @@ def find_one_accel(db):
...
+    query = '''
+         SELECT *
+           FROM tremo;
```
Comment (snelson44) [inline database/database/database.py:2221]: typo

File: database/database/seeds/init_engine_db.sql [hunk 5]
```diff
@@ -58,6 +58,50 @@ CREATE TABLE public.accels (
...
+-- TOC entry ? (class ? OID ?)
+-- Name: tremor; Type: TABLE; Schema: public; Owner: postgres
+--
+
+
+CREATE TABLE public.tremor (
+    id uuid DEFAULT public.gen_random_uuid() NOT NULL,
+    description character varying(255) NOT NULL,
+    patient_id uuid NOT NULL,
+    stationary boolean,
```
Comment (snelson44) [inline database/database/seeds/init_engine_db.sql:91]: Stationary, severity, displacement, and frequency should all have comments attached to them

**Review by snelson44 — CHANGES_REQUESTED** [review]

File: database/database/database.py (hunk 1 above)
Comment (palash-t) [reply database/database/database.py:146]: Done.

File: database/database/database.py (hunk 2 above)
Comment (palash-t) [reply database/database/database.py:711]: Done.

File: database/database/database.py (hunk 3 above)
Comment (palash-t) [reply database/database/database.py:2218]: Done.

File: database/database/database.py (hunk 4 above)
Comment (palash-t) [reply database/database/database.py:2221]: Done.

File: database/database/seeds/init_engine_db.sql (hunk 5 above)
Comment (palash-t) [reply database/database/seeds/init_engine_db.sql:91]: Done.

**Pull request merged**